destination/destination.go

```diff
@@ -138,7 +138,7 @@ func (d *Destination) validateKey(client redis.Conn) error {
 
 // Write receives the record to be written and based on the mode either publishes to PUB/SUB channel
 // or add as key-value pair to stream using XADD, the id of the newly added key is generated automatically
-func (d *Destination) Write(ctx context.Context, rec []sdk.Record) (int, error) {
+func (d *Destination) Write(_ context.Context, rec []sdk.Record) (int, error) {
```

Comment:
Shouldn't this context be used in the `DoContext()` rather?

Reply:
Another great catch :+1: